src/displayapp/screens/Steps.h

```diff
@@ -32,11 +32,14 @@ namespace Pinetime {
         uint32_t currentTripSteps = 0;
 
         lv_obj_t* lSteps;
+        lv_obj_t* lStepsYesterday;
         lv_obj_t* stepsArc;
         lv_obj_t* resetBtn;
         lv_obj_t* resetButtonLabel;
         lv_obj_t* tripLabel;
 
+        static constexpr const char* yesterdayStr = "Yest: %5lu";
```

Comment:
This can move into the .cpp (use an anonymous namespace)

Reply:
It has been banished to the cpp file into an anonymous namespace